cmd/minersc.go

```diff
@@ -68,7 +68,7 @@ var minerscMiners = &cobra.Command{
 		)
 
 		limit, offset := 20, 0
-		active := false
+		active := true
```

Comment:
Please update default flag value in line 627 and 633 as well, otherwise users could get confused.